istioctl/cmd/describe.go

```diff
@@ -1223,6 +1223,11 @@ func containerReady(pod *corev1.Pod, containerName string) (bool, error) {
 			return containerStatus.Ready, nil
 		}
 	}
+	for _, containerStatus := range pod.Status.InitContainerStatuses {
```

Comment:
There should be a little more logic that needs to be handled for the 'describe' command:
https://github.com/istio/istio/blob/2f8eb48b3286308f06f78edfaca0b0ec6edcddae/istioctl/cmd/describe.go#L467-L480

Reply:
thanks